packages/components/src/components/ContextMenu/ContextMenu.module.scss

```
@@ -0,0 +1,54 @@
+.contextMenu {
```

```suggestion
.popover {
```
... oder etwas in der Richtung. Sonst ist es zu nah bei `.contextMenu`